src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/multi-cluster/multi-cluster-form/multi-cluster-form.component.html

```diff
@@ -33,7 +33,7 @@
           <div class="cd-col-form-input">
             <input class="form-control"
                    type="text"
-                   placeholder="https://localhost:4202"
+                   placeholder="https://172.20.0.1:8443"
```

Comment:
172.* is a private IP, but just fyi [IANA reserved example.* for doc purposes](https://en.wikipedia.org/wiki/Example.com).